ztp/gitops-subscriptions/argocd/example/optional-extra-manifest/ipsec/README.md

```diff
@@ -53,10 +47,12 @@ interfaces:
     type: tunnel
 ```
 
-Run the build script to create the MachineConfig:
+Run the build script that creates a MachineConfig to import the external certs and apply the IPsec configuration:
 
 `./build.sh`
 
-Apply the MachineConfig:
+### Deploying IPsec encryption
+
+Include `enable-ipsec.yaml` and `99-ipsec-master-endpoint-config.yaml` files in one of the additional install-time manifests directories defined in the `extraManifests.searchPaths` field in the SiteConfig file.
```

Comment:
Is this solution tested via ZTP? As previous design applies the configs as part of DU profile, testing was done via PGT as day2 configs.

If we want this to be applied during install time, we'd need to retest. @bradyallenjohnson @mcornea

Reply:
Yes, this solution is tested with ZTP by copying both the yaml files to the `siteconfig/sno-extra-manifest` directory, which was specified in the `extraManifestPath` field. I worked together with @sabinaaledort on this solution and we used my ST SNO cluster to test it. The IPSec tunnel came up successfully after ZTP deployment.